docs/appendices/release-notes/unreleased.rst

```diff
@@ -57,6 +57,13 @@ None
 Fixes
 =====
 
+- Fixed a bug that would lead to insertion of records via
```

Comment:
can you wrap this item to 79 chars? you can break apart literal blocks if they have spaces in them no problem